### ISSUE TITLE
Accessibility: V4 config.model.schema '_isEnabled' default value to true

### ISSUE THREAD

**Issue by samumist** [adaptlearning/adapt_framework#2374]
With the V4 release, the default value for `_accessibility.properties._isEnabled.default` in `src/core/schema/config.model.schema` should be set to`true`.

https://github.com/adaptlearning/adapt_framework/blob/master/src/core/schema/config.model.schema#L74
